com/http.go

```diff
@@ -0,0 +1,90 @@
+package com
+
+import (
+	"bytes"
+	"encoding/gob"
+	"fmt"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"serial-data-decryptor/models"
+	"serial-data-decryptor/processor"
+	"serial-data-decryptor/utility"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+)
+
+func StartServer() {
+	fmt.Println("Starting the server")
+
+	ln, err := net.Listen("tcp", utility.GetEnvAsserted("INGRESS_HOST")+":"+utility.GetEnvAsserted("INGRESS_PORT"))
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	fmt.Println("Started the server: " + ln.Addr().String())
+
+	for {
+		conn, err := ln.Accept() // this blocks until connection or error
+		if err != nil {
+			log.Error(err)
+			continue
+		}
+		go handleMessages(conn) // a goroutine handles conn so that the loop can accept other connections
+	}
+}
+
+func handleMessages(conn net.Conn) {
+	dec := gob.NewDecoder(conn)
+	msg := models.Data{}
+	dec.Decode(&msg)
+	fmt.Printf("Received message: %+v\n", msg)
+
+	data, err := processor.Process(msg)
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+	conn.Close()
+
+	//send processed data to egress endpoints
+	sendData(data)
+}
+
+func sendData(data []byte) {
+	egress_Urls := strings.Split(utility.GetEnvAsserted("EGRESS_URLS"), ",")
```

Comment:
`egress_Url` - please use camel case as with the rest of the names

Reply:
stripping is probably needed as well, right? like here: `urls = [url.strip() for url in getenv("EGRESS_URLS").strip(",").split(",")]`

Reply:
yes. Done